server/rpc_test.go

```diff
@@ -270,284 +275,6 @@ func TestRPCSeg(t *testing.T) {
 	o.sessCapErr = nil
 }
 
-func TestEstimateFee(t *testing.T) {
```

Comment:
I see that these tests have been moved - code shuffling is fine if it leads to a more preferable structure, but can we leave that out of this already big PR? Including it here makes it more difficult to spot logical changes that might have occurred, but that are obscured by code blocks being moved around.

Reply:
Yeah you're right. Want me to path mode this out and submit a seperate PR for it?

Reply:
Yeah let's move these changes out of this PR.

Reply:
fixed in 24f0a5c